RatScanner/View/MainWindow.xaml

```diff
@@ -11,6 +11,18 @@
         ResizeMode="CanMinimize"
         Height="400" Width="300">
 
+  <Window.Resources>
+    <Style TargetType="{x:Type Hyperlink}">
+      <Setter Property="TextBlock.FontSize" Value="14" />
+      <Setter Property="Foreground" Value="White" />
+      <Style.Triggers>
+        <Trigger Property="IsMouseOver" Value="True">
+          <Setter Property="Foreground" Value="#569CD6" />
+        </Trigger>
+      </Style.Triggers>
+    </Style>
+  </Window.Resources>
+
```

Comment:
This should probably be a global style for Hyperlinks.
Moving this to `App.xaml` will do the trick.

Reply:
Right, my bad, haven't seen App.xaml

Reply:
Done